### PR TITLE
Study Intro and Week 1

### PR DESCRIPTION
**Pull request by dephillipsmichael**
Many of the Jira issues in this sprint were related so it was hard to separate them out to do independent pull requests.  I went with one PR that includes the study intro screens, and the week 1 home screen.  

Everything should be styled correctly per the new design system and all major functionality should be unit tested.

The week 1 schedules are in place and you can run all the tasks and complete them with the dayOfStudy attached to the upload.  The daily and sleep check-ins are now tasks and not surveys because we had to add an extra field to the schema which is something we cannot do with surveys.

### CODE REVIEW

File: BiomarinPKU/BiomarinPKU-Study/AppDelegate.swift [hunk 1]
```diff
@@ -132,14 +133,32 @@ class AppDelegate: SBAAppDelegate, RSDTaskViewControllerDelegate {
...
         navigator.progressMarkers = []
-        let task = RSDTaskObject(identifier: "signin", stepNavigator: navigator)
+        let task = RSDTaskObject(identifier: onboardingTaskId, stepNavigator: navigator)
         let vc = RSDTaskViewController(task: task)
         vc.delegate = self
         self.transition(to: vc, state: .onboarding, animated: true)
     }
     
+    func studyIntroStep1() -> RSDStep {
+        let step = PKUInstructionStep(identifier: "inrto1")
```
Comment (DanWebster) [inline BiomarinPKU/BiomarinPKU-Study/AppDelegate.swift]: @dephillipsmichael At a surface level, this is a typo, but haven't pulled this town to test if it is not a recognized identifier

File: BiomarinPKU/BiomarinPKU-Study/AppDelegate.swift [hunk 2]
```diff
@@ -132,14 +133,32 @@ class AppDelegate: SBAAppDelegate, RSDTaskViewControllerDelegate {
...
+        step.title = Localization.localizedString("STUDY_INTRO_TITLE_1")
+        step.text = Localization.localizedString("STUDY_INTRO_TEXT_1")
+        step.shouldHideActions = [.navigation(.cancel), .navigation(.goBackward), .navigation(.skip)]
+        step.imageTheme = RSDFetchableImageThemeElementObject(imageName: "Intro1Header")
+        return step
+    }
+    
+    func studyIntroStep2() -> RSDStep {
+        let step = PKUInstructionStep(identifier: "inrto2")
```
Comment (DanWebster) [inline BiomarinPKU/BiomarinPKU-Study/AppDelegate.swift]: @dephillipsmichael same here with typo

**Comment by DanWebster** [comment]
As a followup, saw the same date switching and done reversion on the phone too just for reference

File: BiomarinPKU/BiomarinPKU-Study/AppDelegate.swift (hunk 1 above)
Comment (dephillipsmichael) [reply BiomarinPKU/BiomarinPKU-Study/AppDelegate.swift]: Fixed

**Comment by dephillipsmichael** [comment]
> @dephillipsmichael overall, PR looks fantastic and jam-packed with functionality. This was a fun one to test. On the simulator, I had a few problems with the Week 1 burst home screen changing its state in a weird way. I am logging in with the externalID 1111-1111, so already this may be a culprit because the original registration date for this ID is 9 days ago, but perhaps this is a useful way to probe the code. When I started out before doing an activity, the counter was at day 9/7. Then when I did both checkins, it changed to 1/7. Once I did the physical activity, the daily checkin = done got reverted. A few screenshots for reference.
> 
> And then one last request on this: once an activity has been completed and the check mark is on, can we inactivate the icon so that the participants can't perform a duplicate test?
> 
> Screen Shot 2019-08-13 at 8 54 04 AM
> 
> Screen Shot 2019-08-13 at 8 50 22 AM

This actually was a valid bug.  I scheduled the tasks as persistent so new ones were being made when you completed them and the data was set to today so it was jumping back to day one.  I switched it to only schedule tasks once and it fixed the bug.  You will have to make a new account to see the effects.

**Comment by dephillipsmichael** [comment]
Also fixed the bug where you can do completed tasks @DanWebster

**Comment by dephillipsmichael** [comment]
Also also added fitbit token back in as it now appears to be fixed with the consent fix

**Comment by DanWebster** [comment]
Haven't had a chance yet to test with a new participantID, but want to get all of this merged in and we can add on if anything was awry.

**Comment by dephillipsmichael** [comment]
> Haven't had a chance yet to test with a new participantID, but want to get all of this merged in and we can add on if anything was awry.

Sounds good.  I think it should be ready for a release.

**Pull request merged**